src/database/migrations/20200312122638-create-request.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
       allowNull: false,
       type: Sequelize.STRING
     },
-    travelTo: {
+    destination: {
```

Comment:
Why did you change this attribute? 
Could you keep it as `travelTo` and when you are using it in a query you put it in quotes for eg. `SELECT "travelTo" FROM ...`?

Reply:
@the22mastermind Thank you so much for this suggestion of yours, I implemented it and it is working, without changing other people's work, thank you so much, I didn't know that I'd use `"travelTo"`, because using `travelTo` was throwing an error, but now is okay! Thank you a lot my brother

Reply:
You're welcome bro.